### PR TITLE
Add GitHub issue templates for bugs, features, support and PRs

### PR DESCRIPTION
**Pull request by dragetd**
Issues are either a bug report, feature request or support request. Labels
are automaticaly added to the issues respectively: bug, enhancement, support.

The templates suggest using the Slack before opening an issue, but no longer
say it is a hard requirement. This might give a more inclusive impression to
people reporting who are not Slack users.

- [x] a conversation was held in the appropriate [SDKMAN Slack](https://slack.sdkman.io) channel.
- [x] a Github Issue was opened for this feature / bug.
- [ ] test coverage was added (Cucumber or Spock as appropriate).

This addresses Issue #753

CONTRIBUTING.md was also rephrased.

The tag 'support' would be new. We could not set a tag there. Or even rephrase the text to be more 'We do not want to offer support via GitHub AT ALL', if requested.

I have set it to the default branch of my fork, so you can test how it looks via this repository: https://github.com/draget-forks/sdkman-cli (feel free to create test-issues or PRs in there!)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<!-- Thank you for using the SDKMAN! issue tracker! If you are unsure if it is a bug, then consider using the Slack chat before creating a new issue. Please understand, that we cannot offer regular support through GitHub issues and might close the issue, if it is not an actual bug. -->
+
+**Bug report**
+<!-- A clear and concise description of the bug you encountered  -->
+
+**To reproduce**
+<!-- Steps to reproduce the behavior or verify the issue -->
+
+**System info**
+<!-- Please add relevant information about your system, e.g. Win/Linux, OS version, (WSL?), etc. Please also include the output of `sdk version` -->
```
Comment (marc0der) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: Also important is the shell (zsh/bash) and its version.

File: .github/ISSUE_TEMPLATE/support_request.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Please consider the following things. Note, that ignoring all of these might lead to the issue simply being closed. -->
+
+- [ ] I have checked [the documentation](https://sdkman.io/usage)
+- [ ] I have done a quick search in [past issues](https://github.com/rclone/rclone/issues?q=) or [Stack Overflow](https://stackoverflow.com/questions/tagged/sdkman) for similar problems
+- [ ] I have brought up a conversation in the [Slack User Issues Channel](https://sdkman.slack.com/app_redirect?channel=user-issues)
+
+**Question**
+<!-- A clear and concise description of the issue you are encountering -->
+
+**System info**
```
Comment (marc0der) [inline .github/ISSUE_TEMPLATE/support_request.md:20]: As above, the shell info will also be useful here.

**Comment by dragetd** [comment]
I rebased and updated the branch. The new template can be checked here: https://github.com/draget-forks/sdkman-cli/issues/new/choose

I'd say it is worth a try. If it creates more confusion than it helps, it can be easily reverted. Thank you for the constructive discussion!

**Comment by marc0der** [comment]
Thanks for contributing these!

**Pull request merged**